src/components/Navbar.js

```diff
@@ -20,6 +21,7 @@ const Navbar = () => (
           </li>
         </ul>
       </div>
+      <div className="user-icon"><MdPerson className="user" /></div>
```

Comment:
- [ ] It would be better to add a bottom border to the nav bar as in the Design in order to match it 
- Yours

<img width="1439" alt="Screenshot 2023-05-19 at 16 35 20" src="https://github.com/Bellagirl-maker/book-store/assets/61790587/203a4dba-d7ea-4871-a908-f22375914dbf">

- Design 

<img width="949" alt="Screenshot 2023-05-19 at 16 36 11" src="https://github.com/Bellagirl-maker/book-store/assets/61790587/648c181d-3a27-4525-8aa4-c2a53f0be163">

Reply:
@youmari Done with this change